usaspending_api/search/v2/views/spending_by_award.py

```diff
@@ -98,12 +120,20 @@ def post(self, request):
             "sort_order": json_request["order"],
             "upper_bound": json_request["page"] * json_request["limit"] + 1,
         }
+        self.elasticsearch = is_experimental_elasticsearch_api(request)
+        self.last_id = json_request.get("last_id")
+        self.last_value = json_request.get("last_value")
```

Comment:
Won't these only be provided if the user is sending the experimental elasticsearch flag?

Reply:
yeah. that's why they use `get`

Reply:
Not super dangerous (particularly since this is an intermediary state), but I would rather see these in the conditional below line 132 to ease the distinction between workflows.

Reply:
sure